src/Core/Resolvers/CosmosMutationEngine.cs

```diff
@@ -49,6 +54,7 @@ private async Task<JObject> ExecuteAsync(IMiddlewareContext context, IDictionary
 
             if (string.IsNullOrEmpty(dataSourceName))
```

Comment:
Reuse function similar to GetValidatedDataSourceName in CosmosQueryEngine?

Reply:
Removed the function itself, while doing a minor refactor.